persistence/javadsl/src/main/scala/com/lightbend/lagom/internal/javadsl/persistence/ReadSideActor.scala

```diff
@@ -53,11 +66,11 @@ private[lagom] object ReadSideActor {
  * Read side actor
  */
 private[lagom] class ReadSideActor[Event <: AggregateEvent[Event]](
-    tagName: String,
+    workerCoordinates: WorkerCoordinates,
```

Comment:
This was a pending improvement: having both the projection name and the tag name inside the actor. 
The need to report the projection name in the metrics finally twisted my arm into implementing it.